src/pages/workspaces/workspace/WorkspaceMenu.test.js

```diff
@@ -1,12 +1,11 @@
-import { fireEvent, render, screen } from '@testing-library/react';
+import { fireEvent, screen } from '@testing-library/react';
 import { axe } from 'jest-axe';
-import { useState } from 'react';
 import { div, h } from 'react-hyperscript-helpers';
 import { MenuTrigger } from 'src/components/PopupTrigger';
-import TooltipTrigger from 'src/components/TooltipTrigger';
 import { useWorkspaceDetails } from 'src/components/workspace-utils';
 import * as WorkspaceUtils from 'src/libs/workspace-utils';
 import WorkspaceMenu, { tooltipText } from 'src/pages/workspaces/workspace/WorkspaceMenu';
+import { renderWithAppContexts as render } from 'src/testing/test-utils';
```

Comment:
Since this test is no longer mocking TooltipTrigger (see next comment), it needs a theme context for rendering tooltips.